### PR TITLE
Move tags and all tag related action to v2 API (fetch)

### PR DESCRIPTION
**Pull request by stevieing**
Closes #

#### Changes proposed in this pull request

- remove tags from vuex store
- remove unused tags method from Pinia store.
- removed traction tags data as it was unused
- move tags to v2
- improve pacbio and ont tag set factories
- add pacbio and ont tag set factories to e2e tests
- add printer factory to e2e tests to remove fetch failed error messages
- removed fixtures for ont and pacbio tag sets
- removed fixtures for printers
- tried wherever possible to reduce brittleness of tests which are using ids.

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: src/store/traction/mutations.js [hunk 1]
```diff
@@ -1,9 +1,6 @@
 import defaultState from './state'
 
 const mutations = {
-  setTags(state, tags) {
-    state.tractionTags = tags
-  },
```
Comment (seenanair) [inline src/store/traction/mutations.js:6]: Do we no longer require setTags?

File: src/store/traction/getters.js [hunk 2]
```diff
@@ -1,6 +1,4 @@
 const getters = {
-  tagsRequest: (state, getters, rootState) => rootState.api.v1.traction.tags,
-  tractionTags: (state) => state.tractionTags,
```
Comment (seenanair) [inline src/store/traction/getters.js:3]: Just to understand - are we not keeping tags in store anymore?

**Review by seenanair — COMMENTED** [review]
Looks good. 
I have asked only a few questions related to tags removed from store.

File: src/store/traction/mutations.js (hunk 1 above)
Comment (stevieing) [reply src/store/traction/mutations.js:6]: All tag related stuff has moved to v2. This is all in Pinia store. This is no longer required.

File: src/store/traction/getters.js (hunk 2 above)
Comment (stevieing) [reply src/store/traction/getters.js:3]: Same as above.

**Review by seenanair — APPROVED** [review]

**Pull request merged**